WcaOnRails/Gemfile

```diff
@@ -1,14 +1,20 @@
 # frozen_string_literal: true
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.1'
+# From https://github.com/bundler/bundler/issues/4978#issuecomment-272248627
```

Comment:
@lgarron, this one's for you =) Gets rid of those annoying "insecure git connection" warnings.